dataland-frontend/src/components/forms/CreateGdvDataset.vue

```diff
@@ -0,0 +1,343 @@
+<template>
+  <Card class="col-12 page-wrapper-card p-3">
+    <template #title>New Dataset - GDV</template>
+    <template #content>
```

Comment:
![image](https://github.com/d-fine/Dataland/assets/104447806/b31ef595-30f0-4625-b03f-4b1def827820)
Toggle button to hide empty data does not seem to work on the view page

Reply:
I don't think that this is an issue with our code.  
I think that the "showIf" is responsible for this.  It resolves to `true` for a lot of fields, as soon as `berichtsPflicht` is set to "Yes".    So I don't see no action on our side here.

Reply:
I dont think it will be part of this story, but i feel that the toggle button should be modified in such a way that also the empty fields here are effected. Maybe we can have a short team discussion about this

Reply:
Create follow up story to increase user conveninence on the view page to make the toggle button toggle all fields and not only a few of them

Reply:
Follow up was created https://jira.d-fine.dev/browse/DALA-3184 . Closing this here